test/NuGet.Clients.Tests/NuGet.VisualStudio.Common.Test/Telemetry/PackageSourceTelemetryTests.cs

```diff
@@ -144,22 +198,21 @@ public void AddAggregateData_CountsBytes()
             // Act
             for (int i = 0; i < bytes.Length; i++)
             {
-                var pde = CreateSampleProtocolDiagnosticsEvent(bytes: bytes[i]);
-                PackageSourceTelemetry.AddAggregateData(pde, data);
+                var pde = CreateSampleHttpEvent(bytes: bytes[i]);
+                PackageSourceTelemetry.AddHttpData(pde, data);
             }
 
             // Assert
             var pair = Assert.Single(data);
-            var nupkgData = pair.Value.Nupkg;
-            Assert.Equal(bytes.Sum(), nupkgData.TotalBytes);
-            Assert.Equal(bytes.Max(), nupkgData.MaxBytes);
+            var httpData = pair.Value.Http;
+            Assert.Equal(bytes.Sum(), httpData.TotalBytes);
         }
 
         [Fact]
-        public void AddAggregateData_Counts()
+        public void AddHttpData_MultipleEvents_CountsBools()
```

Comment:
This entire test feels weird to me. trueFalseList is only ever 1 true and 1 false? So what's the point of all the looping and division by 2. Not sure about the maintainability of this one, and it's certainly not intuitive to me, anyway.

Reply:
discussed offline